autoPyTorch/utils/results_visualizer.py

```diff
@@ -71,8 +71,7 @@ def extract_dicts(
         return colors, labels
 
 
-@dataclass(frozen=True)
-class PlotSettingParams:
+class PlotSettingParams(NamedTuple):
```

Comment:
This change is due to the fact that `dataclass` does not allow to have `dict` or `list`.